hil/cli/network.py

```diff
@@ -0,0 +1,81 @@
+"""Commands related to networks are in this module"""
+import click
+import sys
+from hil.cli.client_setup import setup_http_client
+
+C = setup_http_client()
+
+
+@click.group()
+def network():
+    """Commands related to network"""
+
+
+@network.command(name='create', short_help='Create a new network')
+@click.argument('network')
+@click.argument('owner')
+@click.option('--access')
+@click.option('--net_id')
```

Comment:
@jeremyfreudberg thinks that the optional arguments should be shown without typing`--help` explicitly.